### ISSUE TITLE
Installation Error - Eclipse Mars

### ISSUE THREAD

**Issue by Kurt-E-Clothier** [acanda/eclipse-pmd#31]
When trying to install eclipse-pmd, I get the following error:

> Artifact not found: 
> http://www.acanda.ch/eclipse-pmd/release/latest/content.jar.
> http://www.acanda.ch/eclipse-pmd/release/latest/content.jar

This same problem occurs when drag-drop installing from the Eclipse Marketplace or going through *Help > Install New Software*

Eclipse Info:
- Eclipse IDE for Java Developers
- Version: Mars.1 Release (4.5.1)
- Build id: 20150924-1200

**Comment by acanda** [comment]
Unfortunately I can't reproduce this issue. Are you using a proxy? The only thing I currently can come up with is that something went wrong the first time your proxy tried to download content.jar resulting in a corrupt file which is now cached in your proxy.

You could try the update site `http://www.acanda.ch/eclipse-pmd/release/1.7`. This is the exact same thing as `http://www.acanda.ch/eclipse-pmd/release/latest` but under a different URL. This would force your proxy to download the files again.

**Comment by Kurt-E-Clothier** [comment]
Thank you for the help. I was able to download the JAR. Can that be put directly in the Eclipse "dropins" directory? I will hang on to that for now, in case I have issues in the future.

 I am using Google Chrome, no Proxy. I was tethering over USB for internet from my phone, switched to mobile hotspot, and it still didn't work. I uninstalled and reinstalled Eclipse a few times using the OOmph installer, and tried to download a complete Eclipse package. When that didn't work, I tried the OOmph installer again, and this last time I was able to install PMD by dragging the "Install" button link directly into the workspace without any errors. 

It is working fine now. It does sound like some sort of persistent configuration file issue, but I don't really understand what the real problem was. I will close this issue. Thanks again!
